lib/modules/manager/bazel-module/parser.ts

```diff
@@ -11,29 +16,54 @@ const supportedRules = [
   'git_override',
   'local_path_override',
   'single_version_override',
+  ...bzlmodMavenMethods,
 ];
 const supportedRulesRegex = regEx(`^${supportedRules.join('|')}$`);
 
 /**
  * Matches key-value pairs:
  * - `name = "foobar"`
+ * - `name = True`
+ * - `name = ["foo", "bar"]`
  **/
 const kvParams = q
   .sym<Ctx>((ctx, token) => ctx.startAttribute(token.value))
   .op('=')
   .alt(
     q.str((ctx, token) => ctx.addString(token.value)),
     q.sym<Ctx>(booleanValuesRegex, (ctx, token) => ctx.addBoolean(token.value)),
+    q.tree({
+      type: 'wrapped-tree',
+      maxDepth: 1,
+      startsWith: '[',
+      endsWith: ']',
+      postHandler: (ctx) => ctx.endArray(),
+      preHandler: (ctx) => ctx.startArray(),
+      search: q.many(q.str<Ctx>((ctx, token) => ctx.addString(token.value))),
+    }),
   );
 
 const moduleRules = q
-  .sym<Ctx>(supportedRulesRegex, (ctx, token) => ctx.startRule(token.value))
+  .opt<Ctx>(
+    q.sym(mavenVariableRegex, (ctx, token) => {
+      return ctx.startRule(token.value);
+    }),
+  )
+  .opt(q.op('.'))
```

Comment:
Maven-specific matcher should be extracted into `mavenRules` and composed with the `moduleRules` via `alt()` (see `const rule = q.alt<Ctx>(moduleRlues);`).